osu.Game/Overlays/BeatmapSet/Header.cs

```diff
@@ -53,9 +53,12 @@ public BeatmapSetInfo BeatmapSet
                 artist.Text = BeatmapSet.Metadata.Artist;
                 onlineStatusPill.Status = BeatmapSet.OnlineInfo.Status;
 
-                downloadButtonsContainer.FadeIn();
-                noVideoButtons.FadeTo(BeatmapSet.OnlineInfo.HasVideo ? 0 : 1, transition_duration);
-                videoButtons.FadeTo(BeatmapSet.OnlineInfo.HasVideo ? 1 : 0, transition_duration);
+                if (beatmaps.QueryBeatmapSet(s => s.OnlineBeatmapSetID == beatmapSet.OnlineBeatmapSetID) == null)

```

Comment:
doesn't this mean they will never fade out if they are already visible?

Reply:
hm is there a particular scenario you have in mind? i haven't come across one where they do since they are faded out on set added and checked for here on set changed